docs/conf.py

```diff
@@ -147,16 +159,6 @@
 # Integration of notebooks
 gammapy_sphinx_notebooks(setup_cfg)
 
-
-# Theme style
-# html_style = ''
-def setup(app):
```

Comment:
@cdeil Is there a reason for this "ancient" loading of JS and CSS files?

Reply:
@joleroi 

* `gammapy.css` is the main stylesheet for the docs webpage
* `gammapy.js` display the warning box for old version docs
* `copybutton.js` provides the functionality enable/disable >>> in the python code blocks of the docs

Reply:
@Bultako - I think @joleroi preserved these CSS and JS files, he just moved them from `def setup(app):` function to the `html_css_files` and `html_js_files` lists above. Presumably that's the more modern way to do it, and the other way is from years ago? So OK to merge?

Reply:
Ok, I misunderstood the question :)

Reply:
Exactly @cdeil, app.add_stylesheet() and app.add_javascript() are [deprecated](https://www.sphinx-doc.org/en/master/changes.html) since version 1.8.0b1.